.github/ISSUE_TEMPLATE/report_a_bug.yml

```diff
@@ -0,0 +1,144 @@
+name: 🐛 Report a bug
+description: Have you found a bug or issue? Create a bug report for OpenFGA
+labels: [ "bug" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: markdown
+    attributes:
+      value: |
+        **Please do not report security vulnerabilities here**. Use https://github.com/openfga/openfga/security/advisories/new or send us an email at security@openfga.dev instead.
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      options:
+        - label: I have looked into the [README](https://github.com/openfga/openfga/blob/main/README.md) and have not found a suitable solution or answer.
+          required: true
+        - label: I have looked into the [documentation](https://openfga.dev/docs) and have not found a suitable solution or answer.
+          required: true
+        - label: I have searched the [issues](https://github.com/openfga/openfga/issues) and have not found a suitable solution or answer.
+          required: true
+        - label: I have upgraded to the [latest version](https://github.com/openfga/openfga/releases/latest) of OpenFGA and the issue still persists.
+          required: true
+        - label: I have searched the [Discord](https://discord.gg/8naAwJfWN6) community and have not found a suitable solution or answer.
+          required: true
```

Comment:
not a blocker for this PR, but I wouldn't like to encourage the community to go through Discord. That's _a lot_ of messages.